cloudflare/src/framework/apiclient.rs

```diff
@@ -16,4 +16,14 @@ pub trait ApiClient {
         ResultType: ApiResult,
         QueryType: Serialize,
         BodyType: Serialize;
+
+    /// Block and send a request to the Cloudflare API, get the response as bytes.
+    ///
+    fn request_binary<QueryType, BodyType>(
```

Comment:
for me, this puts the decision of whether a request should expect binary or not in the wrong place; as a user i think i should still be simply using `request`, and it should be up to the endpoint developers to specify what should come back, but this interface suggests that i might be able to call either request or request binary on any endpoint and that isn't so. an endpoint should be considered one or the other, but the method request might be better off taking an enum...